recipes/cuda-nvml-dev/build.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# Install to conda style directories
+[[ -d lib64 ]] && mv lib64 lib
+mkdir -p ${PREFIX}/lib
+[[ -d pkg-config ]] && mkdir -p ${PREFIX}/lib && mv pkg-config ${PREFIX}/lib/pkgconfig
+[[ -d "$PREFIX/lib/pkgconfig" ]] && sed -E -i "s|cudaroot=.+|cudaroot=$PREFIX|g" $PREFIX/lib/pkgconfig/nvidia-ml-*.pc
+
+[[ ${target_platform} == "linux-64" ]] && targetsDir="targets/x86_64-linux"
+[[ ${target_platform} == "linux-ppc64le" ]] && targetsDir="targets/ppc64le-linux"
+[[ ${target_platform} == "linux-aarch64" ]] && targetsDir="targets/sbsa-linux"
+
+for i in `ls`; do
+    [[ $i == "build_env_setup.sh" ]] && continue
+    [[ $i == "conda_build.sh" ]] && continue
+    [[ $i == "metadata_conda_debug.yaml" ]] && continue
+    if [[ $i == "lib" ]] || [[ $i == "include" ]]; then
+        # Headers and libraries are installed to targetsDir
+        mkdir -p ${PREFIX}/${targetsDir}
+        mkdir -p ${PREFIX}/$i
+        cp -rv $i ${PREFIX}/${targetsDir}
+        # Nothing to be symlinked in $PREFIX/lib
+    else
+        # Skip all other files (only samples here)
+        # mkdir -p ${PREFIX}/${targetsDir}/${PKG_NAME}
+        # cp -rv $i ${PREFIX}/${targetsDir}/${PKG_NAME}
+    fi
```

Comment:
CI is failing here. I think you need a no-op to make the "else...fi" happy. https://unix.stackexchange.com/questions/133972/bash-syntax-error-when-else-follows-an-empty-then-clause
```suggestion
    else
        # Skip all other files (only samples here)
        :
        # mkdir -p ${PREFIX}/${targetsDir}/${PKG_NAME}
        # cp -rv $i ${PREFIX}/${targetsDir}/${PKG_NAME}  
    fi
```